website/docs/getting-started.mdx

```diff
@@ -46,7 +46,60 @@ Steps:
 
 ## 1. Create a configuration file.
 
-CSpell can use JSON, Yaml, and JavaScript files for configuration. It automatically searches for one of the following: `cspell.json`, `cspell.config.yaml`, `cspell.config.cjs`.
+CSpell can use JSON, Yaml, and JavaScript files for configuration. It automatically searches for one of the following:
+
+{/* * @see https://github.com/streetsidesoftware/cspell/blob/main/packages/cspell-lib/src/lib/Settings/Controller/configLoader/configLocations.ts */}
```

Comment:
Just noticed that prettier inserted a space, converting the jsdoc comment to an odd-looking regular comment

Reply:
prettier should be turned off for .mdx files. There are a lot of bugs in prettier related to .mdx: https://github.com/prettier/prettier/issues?q=is%3Aissue%20state%3Aopen%20mdx

Reply:
Done deal in https://github.com/streetsidesoftware/cspell/pull/7094/commits/5a50b4c48cdbb03c0b67fc2699ac199108de2cce